packages/matter.js/src/cluster/server/AttributeServer.ts

```diff
@@ -189,11 +188,12 @@ export abstract class BaseAttributeServer<T> {
     validateWithSchema(value: T) {
         try {
             this.schema.validate(value);
-        } catch (error) {
-            if (error instanceof ValidationError) {
-                error.message = `Validation error for attribute "${this.name}"${error.fieldName !== undefined ? ` in field ${error.fieldName}` : ""}: ${error.message}`;
-            }
-            throw error;
+        } catch (e) {
+            ValidationError.accept(e);
```

Comment:
Thats not right ... error is thrown in any case - just the message is changed when it is a ValidationError!

Reply:
I think this is handled the same way now.  `ValidationError.accept` rethrows if it's not a validation error.  If it is then the next line changes the message then the following one throws.  Maybe `accept` is a confusing name?  Could maybe be `ValidationError.rethrowOthers(e)`...

Reply:
right